src/CachePlugin.php

```diff
@@ -194,11 +231,73 @@ private function getMaxAge(ResponseInterface $response)
     private function configureOptions(OptionsResolver $resolver)
     {
         $resolver->setDefaults([
+            'cache_lifetime' => 2592000, // 30 days
```

Comment:
might be clearer to actually have an arithmetic expression here since it's only executed once